### ISSUE TITLE
Pairing Failure Xcode Logs

### ISSUE THREAD

**Issue by kristencgm** [LoopKit/Loop#950]
**Describe the bug**
This is a continuation of today's earlier FB post that you replied to about a 722 pump pair error with v2.0E.

**Attach an Issue Report**
Tap the Loop settings icon on the bottom of the screen, then tap Issue Report and attach it to this ticket.

[722CAL Loop Issue Report May 8 2019.pdf](https://github.com/LoopKit/Loop/files/3157981/722CAL.Loop.Issue.Report.May.8.2019.pdf)

**To Reproduce**

**Expected behavior**
Loop app to connect to pump during app setup process.

**Screenshots**

**Phone**
 - Hardware: [e.g. iPhone 8+]
 - OS Version: [e.g. iOS 12.2]

**Loop Version**
 - Version Number: v1.9.5
 - Repo: Loop-master - from https://loopkit.github.io/loopdocs/build/build-loop/

**CGM**
 - Device: Dexcom G5]
 - Manager app: Dexcom App

**Pump**
 - Manufacturer: Medtronic
 - Model:  722
 - Firmware version: 2.0E 1.1 0B 0B]

**Additional context**

 Here's the URL to the Xcode debugging log gist: https://gist.github.com/kristencgm/b5f87fac8c2258cd21f2b0afce4a7e73

FYI - I connected the iPhone, rebuilt the Loop app and tried to connect the MDT pump 3 times while still connected to create the Xcode debugging log.

**Comment by ps2** [comment]
Thanks. This is interesting.  The pump seems to be having a problem with the setRemoteControlEnabled command, but is otherwise responding to radio control just fine.  I'm guessing the pump is usable, but we'd need to add code to support this kind of behavior. 

Could you go into the pump UI, go to Utilities -> Remote Options -> On -> Review ID and tell us what it says there?

**Comment by kristencgm** [comment]
> Thanks. This is interesting. The pump seems to be having a problem with the setRemoteControlEnabled command, but is otherwise responding to radio control just fine. I'm guessing the pump is usable, but we'd need to add code to support this kind of behavior.
> 
> Could you go into the pump UI, go to Utilities -> Remote Options -> On -> Review ID and tell us what it says there?

There are three IDs in the pump (I copied my menus exactly because mine works).
Here's what the screen says:
1  887844
2  886416
3  897464

We also tried with just one ID that read "1  000000", but that didn't seem to make a difference.

**Comment by ps2** [comment]
Can you delete all of them, and try?

**Comment by kristencgm** [comment]
> Can you delete all of them, and try?

Sure. I can do that. Just for clarity sake, though, should I delete them all and have no IDs listed in the pump? Or try with just one 'OOOOOO' ID again?

**Comment by kristencgm** [comment]
> Can you delete all of them, and try?
Deleted all and entered an ID of 000000 in pump.
Deleted and rebuilt Loop app on phone and tried, again.
A couple times I ran it with no ID, but figured out that with no ID the remote option was off so I entered the 000000 ID. Then,  a couple times it detected my pump (didn't clear the area fast enough). Here's an updated Loop issue report:

[Loop Issue Report.pdf](https://github.com/LoopKit/Loop/files/3158906/Loop.Issue.Report.pdf)

Here's the link to the new Xcode debugging log gist: https://gist.github.com/kristencgm/348c6812d8e85f49fa0c10bf7d03132d
